src/pages/dataElements/List.tsx

```diff
@@ -51,8 +52,16 @@ export const Component = () => {
 
     useSectionListParamsRefetch(refetch)
 
+    const [selected, setSelected] = useState<{ value: string; label: string }>()
+
```

Comment:
I can handle this in another PR - but we should integrate this with the `queryParams`, so it's stored in the url. Like the other filters.